src/main/resources/resources/logisim/strings/std/std_pl.properties

```diff
@@ -710,6 +707,9 @@ stdLabelLocAttr = Lokalizacja etykiety
 stdLabelVisibility = Etykieta Widoczna
 stdLogisimEvolutionAppearance = Logisim-Evolution
 stdTriggerAttr = Trigger
+stdSelectLocAttr = wybierz lokalizację
+# ==> stdSelectBottomLeftOption =
```

Comment:
@MarcinOrlowski

Reply:
The translation is OK, however it should start with capital `W`. I see that PL translation file is inconsistent and has other problems (like i.e. incorrect capitalization in `stdLabelVisibility`), `stdTriggerAttr` is not translated etc. It is quite likely that it "survived" several cleanups I've made in meantime because It was deleted and then restored. I will create a ticket on myself to review that file. 

Anyway [prop-tool v2.0.0](https://github.com/marcinOrlowski/prop-tool) is almost here, so that could help keeping translations in good shape.